dask_expr/reductions.py

```diff
@@ -2,6 +2,9 @@
 import toolz
 from dask.dataframe.core import (
     _concat,
+    idxmaxmin_agg,
```

Comment:
Are we ok with dispatching like this?

Reply:
You mean using dask.dataframe code?  If so, yes.  In some future swap we'll bring all of these functions into this library.

Reply:
I've been rewriting overly-complex things that have been about how we define algorithms, but leaving functions-used-as-tasks alone.  At this stage I care a lot about how we organize operations and very little about specific execution code.  

So for example I really enjoyed rewriing the ApplyConcatApply logic (this is used in reductions, groupby-aggregations, and more).

Not sure if that makes sense or not.

Reply:
No that helps, thx for the explanation. Makes judging stuff easier